frame/identity/src/benchmarking.rs

```diff
@@ -91,7 +97,7 @@ benchmarks! {
 	// These are the common parameters along with their instancing.
 	_ {
 		let r in 1 .. MAX_REGISTRARS => add_registrars::<T>(r)?;
-		let s in 1 .. T::MaxSubAccounts::get() => {
+		let s in 1 .. T::MaxSubAccounts::get() - 1 => {
```

Comment:
Why `-1`?

Reply:
Because `set_subs` do one more (and then we get an error) and to keep a common range. Another option may be to remove it here and add it only at `set_subs` benchmark.

Reply:
Mmm, looks like the check is wrong `ensure!(subs.len() <= T::MaxSubAccounts::get() as usize, Error::<T>::TooManySubAccounts);`, should be `>` according to the docs: `/// The maximum number of sub-accounts allowed per identified account.`

Reply:
why does `set_subs` do one more?

Reply:
I don't see a (strong) reason to push `n + 1`. The test was written in that way before the pr. I'll change it to push `n`.

Reply:
Yeah seems like just an oversight or something